core/EE_Dependency_Map.core.php

```diff
@@ -953,7 +957,6 @@ protected function _register_core_aliases()
             'EventEspresso\core\services\loaders\LoaderInterface'                          => 'EventEspresso\core\services\loaders\Loader',
             'CommandFactoryInterface'                                                      => 'EventEspresso\core\services\commands\CommandFactoryInterface',
             'EventEspresso\core\services\commands\CommandFactoryInterface'                 => 'EventEspresso\core\services\commands\CommandFactory',
-            'EventEspresso\core\domain\services\session\SessionIdentifierInterface'        => 'EE_Session',
```

Comment:
I added an `is_subclass_of()` check to `core/services/loaders/ClassInterfaceCache.php` to verify that aliases being added there were indeed a class that can safely be substituted for the supplied FQCN. The session does not load until the WP `init` action so this alias can not be verified until then, therefore it should not be added until that time. In other words: adding the alias too early caused problems and it can not be used until the session is loaded anyways.
That particular interface however is not actually being used anywhere anymore and does not adequately represent the session class anyways, so it's removal as an alias is sound.